samples/rtl/fladd.py

```diff
@@ -0,0 +1,241 @@
+import dace
```

Comment:
Put a description of what this sample shows

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/11984d3c2c997860d2124ef82133077ccd84e753